src/components/button/Button.stories.js

```diff
@@ -84,7 +84,9 @@ export const Button = {
         ?disabled=${args.disabled}
         size=${args.size}
         iconPosition=${args.iconPosition}
-        description=${args.description}
+        description=${args.destructive
+          ? 'This is a destructive action'
+          : args.description}
```

Comment:
Let's revert these. The application can be responsible for passing in the correct description in this case.

Reply:
ok.

Reply:
Done